src/app/components/operations/register/register.component.ts

```diff
@@ -0,0 +1,99 @@
+import { tokenReference } from '@angular/compiler';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LoginUser } from 'src/app/models/login-user';
+import { NewUser } from 'src/app/models/new-user';
+import { AuthService } from 'src/app/services/auth.service';
+import { TokenService } from 'src/app/services/token.service';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  templateUrl: "./register.component.html",
```